### PR TITLE
[15 minute fix] Fix word wrapping on mobile dashboard

### PR DESCRIPTION
**Pull request by jacobherrington**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

When looking at the dashboard view on my phone (360 * 800), I noticed that articles with longer titles would cause the text in these anchors to start wrapping. Adding this CSS property to the `.crayons-btn` class fixes the issue.

I'm not sure if we want this property on every button on the site, but I **think** we might. @ludwiczakpawel @lisasy please correct me if we shouldn't do that! :sweat_smile: 

## QA Instructions, Screenshots, Recordings
Before:
image

After:
image

## Added tests?

- [ ] Yes
- [x] No, and this is why: minor UI tweak
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: minor UI tweak

### CODE REVIEW

**Review by rhymes — APPROVED** [review]
Love this!

File: app/assets/stylesheets/components/buttons.scss [hunk 1]
```diff
@@ -48,6 +48,7 @@
   cursor: pointer;
   transition: all var(--transition-props);
   border: var(--border-width) solid;
+  overflow-wrap: break-word;
```
Comment (aitchiss) [inline app/assets/stylesheets/components/buttons.scss:51]:
```suggestion
  overflow-wrap: normal;
```

I think this should be 'normal'? I checked locally and it still fixes the issue - according to [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) this makes sure:

> Lines may only break at normal word break points (such as a space between two words).

Which I think is what we'd want for our buttons?

**Comment by jacobherrington** [comment]
@aitchiss Looking at the docs, `normal` does seem to be what we want. Same behavior visually with a lot less opportunity for unintended side-effects. :smile:

**Review by aitchiss — APPROVED** [review]
LGTM 😄 

Funny that `overflow-wrap: normal` isn't the default setting!

**Review by ludwiczakpawel — APPROVED** [review]

**Pull request merged**